"make timing" fails with assertion errors when only initializing some submodules

@mithro :  I deleted skywater-pdk and cloned a fresh copy to check everything as would be seen by the ordinary end user, with everything coming from the Google/SkyWater repository.  I followed your instructions for pulling only the "latest" repository for each standard cell library.  When I do that, though, I still get the version number directories;  they just have nothing in them.  Only "latest" has something in it (and is not a symbolic link).  It appears that "make timing" is seeing those version libraries and expecting to find files in them, and failing with an error when it doesn't find any files there.  Anyway, I am getting errors like "AssertionError: libraries/sky130_fd_sc_hd/v0.0.2" (there is such a directory, but it is empty because I did not initialize that submodule).

--- a/Makefile
+++ b/Makefile
@@ -69,14 +69,18 @@ LIBRARIES = $(sort $(notdir $(wildcard libraries/sky130_*_sc_*)))
 
 $(LIBRARIES): | $(CONDA_ENV_PYTHON)
 	@$(IN_CONDA_ENV) for V in libraries/$@/*; do \
-		python -m skywater_pdk.liberty $$V; \
-		python -m skywater_pdk.liberty $$V all; \
-		python -m skywater_pdk.liberty $$V all --ccsnoise; \
+		if [ -d "$$V/cells" ]; then \
+			python -m skywater_pdk.liberty $$V; \
+			python -m skywater_pdk.liberty $$V all; \
+			python -m skywater_pdk.liberty $$V all --ccsnoise; \
+		fi \
 	done
 
 sky130_fd_sc_ms-leakage: | $(CONDA_ENV_PYTHON)
 	 @$(IN_CONDA_ENV) for V in libraries/sky130_fd_sc_ms/*; do \
-		python -m skywater_pdk.liberty $$V all --leakage; \
+		if [ -d "$$V/cells" ]; then \
+			python -m skywater_pdk.liberty $$V all --leakage; \
+		fi \
 	done
 
 sky130_fd_sc_ms: sky130_fd_sc_ms-leakage

